Snippets/ASP/ASTP_4/Behaviors.cs

```diff
@@ -83,36 +83,41 @@ public override async Task Invoke(IIncomingLogicalMessageContext context, Func<T
 
 #region BehaviorUsingIProvidePartitionKeyFromSagaId
 
-class OrderIdAsPartitionKeyBehavior : Behavior<IIncomingLogicalMessageContext>
+internal class OrderIdAsPartitionKeyBehavior : Behavior<IIncomingLogicalMessageContext>
```

Comment:
I'm not aware that we use the visibility modifiers in the snippets. Is this something we introduced recently? If yes then we should also do it in the other snippet https://github.com/Particular/docs.particular.net/pull/5158/files#diff-cf2be1fff701b183297b757e57eae615882ad90da0547fc7996623038de7e479L85. If not then we should remove it

Reply:
https://github.com/Particular/docs.particular.net/pull/5158/files#r555686665